core/common/assets/js/utils/helpers.js

```diff
@@ -1,51 +1,60 @@
 export default class Helpers {
 	softDeprecated( name, version, replacement ) {
-		if ( elementorCommon.config.isDebug ) {
-			this.deprecatedMessage( 'soft', name, version, replacement );
-		}
+		elementorDevToolsModule.deprecation.softDeprecated( name, version, replacement );
+
+		elementorDevToolsModule.deprecation.softDeprecated(
+			'elementorCommon.helpers.softDeprecated',
+			'3.7.0',
+			'elementorDevToolsModule.deprecation.softDeprecated'
+		);
 	}
```

Comment:
Like in PHP, we don't have `soft` and `hard` deprecation functions.
```suggestion
		elementorDevToolsModule.deprecation.softDeprecated(
			'elementorCommon.helpers.softDeprecated',
			'3.7.0',
			'elementorDevTools.deprecation.deprecatedFunction'
		);
	}
```